### PR TITLE
Update husky to v6 + pre-commit hook

### PR DESCRIPTION
**Pull request by lebaphi**

### CODE REVIEW

**Comment by IanMayo** [comment]
Hello Phi, thanks for this.

We're getting the ESLINT fixes in a number of branches.

Can I suggest an alternate strategy for this?

Could you branch off `develop` and provide a PR that fixes the ESLINT issues.  When that's merged, those changes will disappear from the PRs, letting us focus on the actual changes.

File: package.json [hunk 1]
```diff
@@ -45,11 +39,14 @@
     "heroku-postbuild": "yarn build && yarn build-storybook",
     "serge": "yarn start-server",
     "snyk-protect": "set NODE_OPTIONS=–max_old_space_size=8192 snyk protect",
-    "prepare": "yarn run snyk-protect",
+    "prepare": "yarn run snyk-protect && husky install",
```
Comment (IanMayo) [inline package.json:42]: In the dev instructions, I don't see a command for `yarn prepare`. Does it get triggered by some other process/call?

Comment (lebaphi) [reply package.json:42]: Hi @IanMayo , this is a special life cycle scripts, more details at https://docs.npmjs.com/cli/v6/using-npm/scripts

**Comment by lebaphi** [comment]
> Hello Phi, thanks for this.
> 
> We're getting the ESLINT fixes in a number of branches.
> 
> Can I suggest an alternate strategy for this?
> 
> Could you branch off `develop` and provide a PR that fixes the ESLINT issues. When that's merged, those changes will disappear from the PRs, letting us focus on the actual changes.

okay, will have a PR to fix that one.

**Comment by IanMayo** [comment]
@lebaphi  - please mark this as `Ready to review` once you're happy with it.

**Comment by IanMayo** [comment]
Hello @lebaphi  - I've just pulled this branch and had a play.

I realise I've mis-described the issue.  I now realise we should be running the tests at `push`, not `commit`.  Running them for each commit is a pain 🤣 

Could you modify the hook to run on pre-push, rather than pre-commit please?

**Comment by lebaphi** [comment]
> Hello @lebaphi - I've just pulled this branch and had a play.
> 
> I realise I've mis-described the issue. I now realise we should be running the tests at `push`, not `commit`. Running them for each commit is a pain 🤣
> 
> Could you modify the hook to run on pre-push, rather than pre-commit please?

Done Ian 👍

**Review by IanMayo — APPROVED** [review]
Updates working great, thanks.

**Pull request merged**